### DISCUSSION TITLE
Always a conflict when not identical

### DISCUSSION THREAD

**Discussion by m-vellinga** [copier-org/copier#456] [Q&A] [answered]
Hi! Loving the idea of easily updating projects that are derived from a template whenever the template changes.

I have a question about the update behaviour. My steps:

- Create a project based on my template, initialise a git repo and commit everything.
- Update some file in the template, commit and tag it
- Run copier update in the project

The file I updated in the template now shows a "conflict" even though that file was never touched in the project itself. (both version 5 and 6 of copier) Is that how it's intended or am I missing something?

**Comment by yajo** [comment]
That shouldn't happen. Sounds like an issue in copier or the template.

**Reply by m-vellinga** [reply]
I setup a small proof of concept

[template](https://github.com/m-vellinga/copier-template-test) at version 0.2.0
[project](https://github.com/m-vellinga/copier-project-test) created from version 0.1.1

Running copier update (version 6.0.0a7) in the project results in:

```
(.venv) ➜  copier-project-test git:(main) cat .copier-answers.yml 
# Changes here will be overwritten by Copier
_commit: v0.1.1
_src_path: gh:m-vellinga/copier-template-test
readme_title: Awesome project

(.venv) ➜  copier-project-test git:(main) copier update

🎤 Something Something
readme_title? Format: str Awesome project
 identical  .
  conflict  .copier-answers.yml
 overwrite  .copier-answers.yml
  conflict  README.md
 Overwrite README.md? [Y/n] 
```

**Comment by yajo** [comment, marked as answer]
I have tested your PoC from https://github.com/copier-org/copier/discussions/456#discussioncomment-1481205.

Maybe the problem is the word "conflict". In this context, a conflict is a file that has to be updated. And that's the case because [the `README.md` has evolved](https://github.com/m-vellinga/copier-template-test/compare/v0.1.1...v0.2.0).

If there were "real" conflicts (things that Copier cannot resolve by itself), you'd see a `README.md.rej` file dropped next to `README.md`.
